cli/app.go

```diff
@@ -618,6 +656,65 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
+					Name:      "tag",
+					Usage:     "tag binary data by filter or ids",
+					UsageText: createUsageText("data tag", nil, true),
+					Subcommands: []*cli.Command{
+						{
+							Name:      "ids",
+							Usage:     "adds or removes tags from binary data by file ids for a given org and location",
+							UsageText: createUsageText("data tag ids", nil, true),
+							Subcommands: []*cli.Command{
+								{
+									Name:  "add",
+									Usage: "adds tags to binary data by file ids for a given org and location",
+									UsageText: createUsageText("data tag ids add", []string{
+										dataFlagTags, generalFlagOrgID,
+										dataFlagLocationID, dataFlagFileIDs,
+									}, false),
+									Flags:  dataTagByIDsFlags,
+									Action: DataTagActionByIds,
+								},
+								{
+									Name:  "remove",
+									Usage: "removes tags from binary data by file ids for a given org and location",
+									UsageText: createUsageText("data tag ids remove", []string{
+										dataFlagTags, generalFlagOrgID,
+										dataFlagLocationID, dataFlagFileIDs,
+									}, false),
+									Flags:  dataTagByIDsFlags,
+									Action: DataTagActionByIds,
+								},
+							},
+						},
+						{
+							Name:      "filter",
+							Usage:     "adds or removes tags from binary data by filter",
+							UsageText: createUsageText("data tag ids", []string{dataFlagTags}, false),
```

Comment:
data tag filter?

Reply:
Gah! Feeling dumb for missing that one 🤦